net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/eap.xml

```diff
@@ -15,7 +15,7 @@
         <id>eap.ca</id>
         <label>Root Certificate</label>
```

Comment:
So this is the client CA ? If so we should substitute Root -> Client in the label and help text? @mimugmail

Reply:
Where is the difference? The Root Certificate from server and client should be the same. Sine the model refers to a CertificateField with CA filter this would be wrong

Reply:
"Root" means top, maybe it's RADIUS terminology here, but as far as I understood it could be any sort of intermediate CA and not the chain up to the root.

> The Root Certificate from server and client should be the same. 

There's nothing that would indicate this is necessarily true and part of the issue @alexanderharm is raising

Reply:
I actually elaborated this in the description (that is the ca that validates client certs), not enough?

Reply:
I guess so, thanks 👍